KubeArmor/feeder/policyMatcher.go

```diff
@@ -1183,8 +1182,7 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 					log.Action = "Audit (Block)"
 				}
 
-				if fd.DefaultPostures[log.NamespaceName].FileAction == "audit" && (secPolicy.Action == "Allow" || secPolicy.Action == "Audit (Allow)") && log.Result == "Passed" {
-					// defaultPosture = audit
```

Comment:
Let's keep this comment.

And also, let's add a commet that we only apply default posture whe log.type isn't yet known.